### PR TITLE
Fix style for system dark

### PR DESCRIPTION
**Pull request by onesounds**
Fixed an issue with the wrong color in dark mode of the Windows. 

## Summary by CodeRabbit

- **Style**
  - Enhanced light and dark mode support for various UI elements including section headers, features, sub-sections, custom sections, footers, modals, buttons, and text.
  - Improved background, border, and fill colors for better visual consistency across themes.
  - Added smoother transitions when switching between light and dark modes for a more polished user experience.

### CODE REVIEW

File: public/style/style.css [hunk 1]
```diff
@@ -53,45 +53,121 @@ TABLE OF CONTENT
...
 .dark-mode {color: var(--text-for-dark); background-color: var(--bkg-for-dark);}
 .dark-mode a {color: var(--link-for-dark);}
 .light-mode {color: var(--text-for-light); background-color: var(--bkg-for-light);}
 .light-mode a {color: var(--text-for-light);}
 
 .light-mode .ui-section-header{ background-color: var(--bkg-header-for-light); }
-.light-mode .ui-section-feature{ background-color: var(--bkg-for-light); border-color: var(--sep-for-light);}
-.light-mode .ui-section-sub{ background-color: var(--bkg-header-for-light); }
 .dark-mode .ui-section-header{ background-color: var(--bkg-header-for-dark); }
+.light-mode .ui-section-feature{ background-color: var(--bkg-for-light); border-color: var(--sep-for-light);}
```
[the remark this answers is not in the document]
Comment (onesounds) [reply public/style/style.css]: @coderabbitai Fixed. Is that ok?

File: public/style/style.css [hunk 2]
```diff
@@ -53,45 +53,121 @@ TABLE OF CONTENT
...
+    }
 }
 
+.ui-component-cta a:hover{
+  border: 1px solid transparent;
+  background: dodgerblue;
+}
 .modal-sep {
   width:100%;
   border : 0;;
```
[the remark this answers is not in the document]
Comment (onesounds) [reply public/style/style.css:173]: I can't find this part. where is it?

File: public/style/style.css [hunk 3]
```diff
@@ -53,45 +53,121 @@ TABLE OF CONTENT
...
 
 .light-mode .card-inside{background-color: var(--card-for-light); border-color: var(--sep-for-light);}
 .dark-mode .card-inside { background-color: var(--card-for-dark); border-color: var(--sep-for-dark);}
-.dark-mode .ui-section-close { background-color: var(--bkg-header-for-dark); border-color: var(--sep-for-dark)}
+.ui-section-footer { background-color: var(--copyright-for-light); border-color: var(--sep-for-light);}
+.ui-section-close { background-color: var(--bkg-header-for-light); border-color: var(--sep-for-light)}
+.dark-mode .ui-section-close { background-color: var(--bkg-header-for-dark); border-color: var(--sep-for-dark)}    .ui-section-footer--copyright p {color: var(--ui-color-typography-note);}
+.ui-section-footer--copyright {color:var(--ui-color-typography-note) !important}
+.details-modal-content-winget {border-top: 1px solid var(--sep-for-light);}
+.light-mode .details-modal-content-winget {border-top: 1px solid var(--sep-for-light);}
```
[the remark this answers is not in the document]
Comment (onesounds) [reply public/style/style.css:88]: fixed. ok?

File: public/style/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (onesounds) [reply public/style/style.css:173]: @coderabbitai review

File: public/style/style.css [hunk 4]
```diff
@@ -53,46 +53,122 @@ TABLE OF CONTENT
...
 .light-mode .ui-section-header{ background-color: var(--bkg-header-for-light); }
-.light-mode .ui-section-feature{ background-color: var(--bkg-for-light); border-color: var(--sep-for-light);}
-.light-mode .ui-section-sub{ background-color: var(--bkg-header-for-light); }
 .dark-mode .ui-section-header{ background-color: var(--bkg-header-for-dark); }
+.ui-section-sub{ background-color: var(--bkg-header-for-light); border-color: var(--sep-for-light); }
+.light-mode .ui-section-sub{ background-color: var(--bkg-header-for-light); border-color: var(--sep-for-light); }
+.light-mode .ui-section-dark{ background-color: var(--bkg-header-for-dark); border-color: var(--sep-for-dark); }
 .dark-mode .ui-section-feature{ background-color: var(--bkg-for-dark); border-color: var(--sep-for-dark);}
 .dark-mode .ui-section-sub{ background-color: var(--bkg-header-for-dark); border-color: var(--sep-for-dark);}
+.ui-section-custom{  background-color: var(--bkg-header-for-light);border-color: var(--sep-for-light);}
```
[the remark this answers is not in the document]
Comment (onesounds) [reply public/style/style.css:71]: ignore that part. and Is there any problem?

**Review by jjw24 — APPROVED** [review]

**Pull request merged**